tests/pipelines/test_api.py

```diff
@@ -226,20 +226,27 @@ def test_get(pipelines_api):
     assert (response['pipeline_id'] == PIPELINE_ID and response['state'] == 'RUNNING')
 
 
-def test_reset(pipelines_api):
```

Comment:
I think it might be good to keep the `test_reset` and `test_run` tests until the run and reset APIs are fully removed.

Reply:
I don't have these methods anymore, cli.py directly calls start_update now. So tests are around start_udpate.